### PR TITLE
updated podspec and changelog for version 1.1

### PR DESCRIPTION
**Pull request by thorstenstark**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 1.1
+* **feature** Migrated to Swift 4.2
+* **improvement** Added more devices to Device
```
Comment (brototyp) [inline CHANGELOG.md:10]: There should be some more according to [this list of commits](https://github.com/tbointeractive/bytes/commits/develop). Do you want to list those here as well?

**Review by brototyp — COMMENTED** [review]
Just a short remark.

**Comment by brototyp** [comment]
Yeah! That looks like you are tackling #17 :) 👍

File: CHANGELOG.md (hunk 1 above)
Comment (thorstenstark) [reply CHANGELOG.md:10]: You're right. I've added those changes.

**Comment by thorstenstark** [comment]
@brototyp yes, it took much too long for this release. Hopefully the next one won't take so long again.

**Review by pstued — APPROVED** [review]
Thanks for the ping and patience 😇

File: bytes.podspec [hunk 2]
```diff
@@ -20,11 +20,11 @@ Pod::Spec.new do |s|
 
   s.license      = "MIT"
 
-  s.author      = { "Cornelius Horstmann" => "horstmann@tbointeractive.com", "Thorsten Stark" => "stark@tbointeractive.com", "Bernhard Eiling" => "eiling@tbointeractive.com", "Pascal Stüdlein" => "stuedlein@tbointeractive.com"  }
+  s.author      = { "Cornelius Horstmann" => "horstmann@tbointeractive.com", "Thorsten Stark" => "stark@tbo.de", "Bernhard Eiling" => "eiling@tbointeractive.com", "Pascal Stüdlein" => "stuedlein@tbointeractive.com"  }
 
   s.platform     = :ios, "8.2"
 
-  s.source       = { :git => "https://github.com/tbointeractive/bytes.git", :tag => 'v1.0'}
+  s.source       = { :git => "https://github.com/tbointeractive/bytes.git", :tag => 'v1.1'}
```
Comment (pstued) [inline bytes.podspec:27]: Just as a reminder. You need to tag it before you can publish it to Cocoapods then

File: bytes/Info.plist [hunk 3]
```diff
@@ -17,7 +17,7 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>0.9</string>
+	<string>1.1</string>
```
Comment (pstued) [inline bytes/Info.plist:20]: 🥇

File: bytes.podspec (hunk 2 above)
Comment (thorstenstark) [reply bytes.podspec:27]: Thanks, good point!

**Pull request merged**